tests/wpt/web-platform-tests/html/syntax/serializing-html-fragments/serializing.html

```diff
@@ -59,7 +59,7 @@
 ["\"", "<span>\"</span>"],
 ["<style><&></style>", "<span><style><&></style></span>"],
 ["<script type=\"test\"><&><\/script>", "<span><script type=\"test\"><&><\/script></span>"],
-["<script type=\"test\"><&><\/script>", "<script type=\"test\"><&><\/script>"],
+["<script><&><\/script>", "<&>"],
```

Comment:
Can I put them both in a single PR? I thought the test being in the servo repository and the serializer being in the html5ever repository made that hard?